README.md

```diff
@@ -1 +1,303 @@
+- Language models ((e.g., Google BERT and LLaMa)
+- [SPARQL](https://www.w3.org/TR/sparql11-query/)
+
+
+## Sequence Diagram
+
+The sequence diagram below shows the interactions between different service components for the KG construction.
+
+
+```mermaid
+sequenceDiagram
+    autonumber
+
+    participant User
+    participant UI
+    participant KG_Construction as (Semi-) structured KG construction
+    participant Mapping as Mapping & Annotation
+    participant Alignment as Alignment & resolution
+    participant Validation as Validation & Quality assurance
+    participant Expert
+    participant Triplestore
+    User->>+ UI: Upload CSV
+    UI->>+KG_Construction: Return response
+    KG_Construction->>+KG_Construction: Perform initial check, e.g., presence of required columns
+
+    alt is invalid
+        KG_Construction-->>+UI: Return Error message
+        UI-->>+User: Return Error message
+    else is valid
+        KG_Construction->>+ Mapping: Perform mapping & annotation as necessary
+        Mapping->>+ Validation: Perform validation of KG triples
+        Validation->>+Validation: Validation checks, e.g., SHACL, provenance conflict
+        Validation->>+ Alignment: Resolve conflicts
+        alt conflict identified, perform resolution
+            Alignment->>+ Alignment: Perform automated conflict resolution and alignment operation
+            Alignment-->>+ Validation: Return response (triples with resolved conflicts)
+            
+
+        else conflict identified, perform resolution-requires human oversight
+            Alignment->>+ Expert: Send to expert for manual conflict resolution
+            Expert-->>+Alignment: Return response (triples with resolved conflicts)
+            Alignment-->>+ Validation: Return response (triples with resolved conflicts) 
+            
+        end 
+            Validation-->>+ Mapping: Return response (validated and conflict resolved KG triples)
+            Mapping-->>+ KG_Construction: Updated KG triples
+            KG_Construction->>+Triplestore: Store KG in database
+            Triplestore-->>+KG_Construction: Return acknowledgement
+            KG_Construction-->>+UI: Return response (operation status notification)
+            UI-->>+User:Send notification
+    end
+```
+
+
+## Hosting Infrastructure
+
+We recognize the current [beta site’s](https://beta.brainkb.org/) issues and are working towards improving it. In particular, we are working on improving the following problems of the beta site.
+
+- **Performance:** There is a performance issue with the beta site's knowledge base page. This is because we are currently using the free version of GraphDB and it only allows two simultaneous queries. Since, fetching the details, such as Library Aliquot and their inter-related information requires running more than two queries, there are performance issues.  We are looking for the premium version of GraphDB and other opensource triple stores as alternatives.
```

Comment:
```suggestion
- **Performance:** We are currently using the free version of GraphDB. It has a limitation of two simultaneous queries. Because fetching the details, such as Library Aliquot and their inter-related information, requires running more than two queries, this limitation impacts the beta site's knowledge base page.  We are currently considering the premium version of GraphDB and other opensource triple stores alternatives.
```

Possibly, you may want to added something like ", and the performance of future version of BrainKB will be significantly improved."

Reply:
Updated